src/CommunityVoices/App/Api/View/Image.php

```diff
@@ -109,4 +109,12 @@ protected function getImageNextImage()
 
         return $response;
     }
+    public function postImageUpdate()
+    {
+
+    }
+    public function postImageUnpair()
+    {
+        
+    }
```

Comment:
Are you sure there's not an error object we should return?  I really think this will return an error.

Reply:
What error would be returned?

Reply:
A response has been added, so resolving.